src/project.cpp

```diff
@@ -1501,7 +1512,13 @@ bool Project::readWildMonData() {
     for (auto subObjectRef : wildMonObj["wild_encounter_groups"].toArray()) {
         QJsonObject subObject = subObjectRef.toObject();
         if (!subObject["for_maps"].toBool()) {
-            extraEncounterGroups.insert(subObject["label"].toString(), subObject);
+            QString err;
+            QString subObjson = QJsonDocument(subObject).toJson();
+            OrderedJson::object orderedSubObject = OrderedJson::parse(subObjson, err).object_items();
```

Comment:
Does this need better error handling?  What `OrderedJson::parse(subObjson, err)` fails?

Reply:
It fails with an invalid object, but I don't expect that would ever happen because I don't think `QJsonDocument::toJson` would return something invalid. I added this check here just in case because it was failing in my tests when I had a Json::string without surrounding it with quotes.